crates/nu-cli/src/commands/classified/external.rs

```diff
@@ -738,7 +738,9 @@ mod tests {
 
         let mut ctx = Context::basic().expect("There was a problem creating a basic context.");
 
-        assert!(run_external_command(cmd, &mut ctx, None, false).is_err());
+        assert!(run_external_command(cmd, &mut ctx, None, false)
+            .await
+            .is_err());
```

Comment:
Honestly, I'm not a huge fan of this formatting, which `rustfmt` is forcing. Should we configure it?

My preferences here would either be:

```rust
assert!(
  run_external_command(cmd, &mut ctx, None, false).await.is_err()
);
```

or 

```rust
assert!(
    run_external_command(cmd, &mut ctx, None, false)
        .await
        .is_err()
);
```